src/render/VolumeRaycastRenderer.cpp

```diff
@@ -200,9 +200,12 @@ void VolumeRaycastRenderer::set_multiplier(float multiplier)
   shader.set_multiplier(multiplier);
 }
 
-void VolumeRaycastRenderer::set_numSlices(int slices)
+void VolumeRaycastRenderer::set_num_slices(int slices)
 {
-  shader.set_stepSize(1.0f / slices, 1.0f / slices, 1.0f / slices);
+  shader.set_stepSize(0.01, 0.01, 0.01);
+  // TO DO:  Camilo Thinks this is not necessary for ray casting volume rendering.
+  // we only need step size. 
+  //shader.set_stepSize(1.0f / slices, 1.0f / slices, 1.0f / slices);
```

Comment:
Add and reference a GitHub issue?

Reply:
https://github.com/brown-ccv/VR-Volumeviewer/issues/59